crates/client/src/app.rs

```diff
@@ -1,68 +1,13 @@
-use yew::{function_component, html, Component, Html, HtmlResult, Suspense};
-use yew_router::{BrowserRouter, Switch};
+use yew::{function_component, html, Html};
+use yew_router::BrowserRouter;
 
-use crate::{
-    account::{dashboard::Dashboard, login::Login, register::Register},
-    route::{self, Route},
-};
+use crate::route::{switch, Route};
 
-#[allow(dead_code)]
```

Comment:
I really don't understand why you don't like this app so much, why do you want to remove that? Or better question where you want to manage basing part of functionality (core)? I know that, made in Client2023 is awful, but still imo we must have something which will be our core.

Reply:
It's only temporary because the app wasn't rendering the /register, /login, and other routes correctly. It only worked when I manually changed the route value. Additionally, the 404 error page was not working.

Reply:
Did you think about staying App that was be, and implementing changing, by e.g. notifying App what is should show basing on `switch` function from router?